### PR TITLE
fix yes to no

### PR DESCRIPTION
**Pull request by michael-grace**

### CODE REVIEW

**Review by ColinRoitt — CHANGES_REQUESTED** [review]
Please format as lowercase 'no' with no punctuation as per style guide

**Comment by michael-grace** [comment]
FINE! But there is no style guide....

File: index.html [hunk 1]
```diff
@@ -56,3 +54,7 @@ <h1>YES!</h1>
     isToday();
 
 </script>
+
+</body>
```
Comment (markspolakovs) [inline index.html:60]: correct html => vonc

**Review by markspolakovs — CHANGES_REQUESTED** [review]

**Review by markspolakovs — CHANGES_REQUESTED** [review]
you still haven't corrected the fact that the html is semantically correct

**Pull request merged**